public/app/features/serviceaccounts/ServiceAccountTable.tsx

```diff
@@ -0,0 +1,227 @@
+import React, { useMemo } from 'react';
+import Skeleton from 'react-loading-skeleton';
+
+import {
+  Avatar,
+  CellProps,
+  Column,
+  InteractiveTable,
+  Pagination,
+  Stack,
+  TextLink,
+  Button,
+  IconButton,
+  Icon,
+} from '@grafana/ui';
+import { UserRolePicker } from 'app/core/components/RolePicker/UserRolePicker';
+import { contextSrv } from 'app/core/core';
+import { AccessControlAction, OrgRole, Role, ServiceAccountDTO } from 'app/types';
+
+import { OrgRolePicker } from '../admin/OrgRolePicker';
+
+type Cell<T extends keyof ServiceAccountDTO = keyof ServiceAccountDTO> = CellProps<
+  ServiceAccountDTO,
+  ServiceAccountDTO[T]
+>;
+
+interface ServiceAccountTableProps {
+  services: ServiceAccountDTO[];
+  onRoleChange: (role: OrgRole, serviceAccount: ServiceAccountDTO) => void;
+  roleOptions: Role[];
+  onRemoveButtonClick: (serviceAccount: ServiceAccountDTO) => void;
+  onDisable: (serviceAccount: ServiceAccountDTO) => void;
+  onEnable: (serviceAccount: ServiceAccountDTO) => void;
+  onAddTokenClick: (serviceAccount: ServiceAccountDTO) => void;
+  showPaging?: boolean;
+  totalPages: number;
+  onChangePage: (page: number) => void;
+  currentPage: number;
+  isLoading: boolean;
+}
+
+export const ServiceAccountTable = ({
+  services,
+  onRoleChange,
+  roleOptions,
+  onRemoveButtonClick,
+  onDisable,
+  onEnable,
+  onAddTokenClick,
+  showPaging,
+  totalPages,
+  onChangePage,
+  currentPage,
+  isLoading,
+}: ServiceAccountTableProps) => {
+  const displayRolePicker =
+    contextSrv.hasPermission(AccessControlAction.ActionRolesList) &&
+    contextSrv.hasPermission(AccessControlAction.ActionUserRolesList);
+  /* @ts-ignore */
```

Comment:
```suggestion

```

This seems to be not necessary. Also it's better to use `@ts-expect-error`, which will give an error when it's not needed anymore.